hlink/linking/model_exploration/link_step_train_test_models.py

```diff
@@ -52,161 +472,123 @@ def _run(self) -> None:
         dep_var = config[training_conf]["dependent_var"]
         id_a = config["id_column"] + "_a"
         id_b = config["id_column"] + "_b"
-        desc_df = _create_desc_df()
+
         columns_to_keep = [id_a, id_b, "features_vector", dep_var]
         prepped_data = (
             self.task.spark.table(f"{table_prefix}training_vectorized")
             .select(columns_to_keep)
             .cache()
         )
 
+        # Stores suspicious data
         otd_data = self._create_otd_data(id_a, id_b)
 
-        n_training_iterations = config[training_conf].get("n_training_iterations", 10)
+        outer_fold_count = config[training_conf].get("n_training_iterations", 10)
+        inner_fold_count = 3
+
+        if outer_fold_count < 3:
+            raise RuntimeError("You must use at least two training iterations.")
```

Comment:
The error message and if statement don't seem to line up here. Do you need at least 2 or at least 3 iterations?

Reply:
Yes, it's three, not two. Fixed.